### ISSUE TITLE
Default FiberFlatNights so MiniSV2 use pre-cleaning data, and are inconsistent with MiniSV2 Data

### ISSUE THREAD

**Issue by akremin** [desihub/desispec#974]
## Breakdown of Nights in andes

- Nights _without_ long cals that **are** impacted by this: [20200227, 20200228, 20200229, 20200303, 20200304]

- Nights _with_ long calibrations that are **not** affected: [20200219, 20200220, 20200223, 20200224, 20200225, 20200226, 20200301, 20200305, 20200308, 20200309, 20200313]

- Nights _without_ long cals that are **not** impacted: [20200314, 20200315]

### MiniSV2 (Problemic)
The current svn tag for DESI_SPECTRO_CALIB has two fiberflatnight-[brz][0-9]-*.fits files for each of the 30 cameras:

One acquired and dated from 20200125, and one dated 2020030[6 or 7]-20200315. 
Therefore all MiniSV2 nights without long cals will use the Jan 25th default data while the SV0 nights will use the March default. 

The problem is that Jan 25 was prior to any spectrographs being cleaned ([DocDB 5666](https://desi.lbl.gov/DocDB/cgi-bin/private/ShowDocument?docid=5666)). 

The reduced MiniSV2 dataset in andes consists solely of data from the six cleaned spectrographs. When comparing cleaned fiberflatnight data from MiniSV2 nights with long cals to these (pre-cleaned) January defaults, there are discrepancies of 10's of percent. 

Below I give two examples comparing Feb 19th (where we have long calibrations), to the defaults in SVN 0.1.1 for two petals, SP0 and SP2 respectively.

![image](https://user-images.githubusercontent.com/8572603/83801925-c2967b80-a65e-11ea-8df9-a644e51a6b97.png)

.

![image](https://user-images.githubusercontent.com/8572603/83803504-8f092080-a661-11ea-9f1a-bcbc64e64f6b.png)

**The trend of large scatter is true across all nights in MiniSV2.** 

### SV0 (Fine)
SV0 data seems to agree better with the default file corresponding to that date range (2020030[6 or 7]-20200315):

![image](https://user-images.githubusercontent.com/8572603/83802210-320c6b00-a65f-11ea-86fb-9bb9f45acc92.png)

.

![image](https://user-images.githubusercontent.com/8572603/83802266-45b7d180-a65f-11ea-8c69-e273d1dce59b.png)

This is true for both March 9th and 13th where long cals were taken.

## Solution

Comparing long cals taken on different nights during MiniSV2 shows percent-level scatter:

![image](https://user-images.githubusercontent.com/8572603/83811434-a7336c80-a66e-11ea-85fa-2bc892599153.png)

.

![image](https://user-images.githubusercontent.com/8572603/83811460-b61a1f00-a66e-11ea-853d-541a3b4c871a.png)

Because the first night in MiniSV2 without long cals was 20200227, the following long-cal nights of MiniSV2 with cleaned spectrographs would be valid for including in the svn repository: [20200219, 20200220, 20200223, 20200224, 20200225, 20200226].

Below I show example plots of those nights for spectrographs 0 and 2:

![image](https://user-images.githubusercontent.com/8572603/83812170-e8784c00-a66f-11ea-8182-d276b45c8938.png)

.

![image](https://user-images.githubusercontent.com/8572603/83812219-fb8b1c00-a66f-11ea-8f8f-b3c75f582185.png)

Any night other than 20200220 (worse variation with wavelength) or 20200225 (missing spectrograph 2) would be reasonable to include in SVN as the default (or a combination of all 4 that are not Feb. 20th or 25th).

**In summary, one of the following (or a combination of the following) should be added to the DESI_SPECTRO_CALIB svn: [20200219, 20200223, 20200224, 20200226].**
